omod/src/main/resources/config.xml

```diff
@@ -126,34 +131,176 @@
+		</description>
+	</globalProperty>
+
+	<globalProperty>
+		<property>${project.parent.artifactId}.mpi.pdqEndpoint</property>
+		<defaultValue>http://localhost:5001/pdq</defaultValue>
+		<description>
+			Specifies endpoint for Hl7 PDQ Message
+		</description>
+	</globalProperty>
+
+	<globalProperty>
+		<property>${project.parent.artifactId}.mpi.pdqPort</property>
+		<defaultValue>3600</defaultValue>
+		<description>
+			Specifies port for Hl7 PDQ Message
+		</description>
+	</globalProperty>
+
+	<globalProperty>
+		<property>${project.parent.artifactId}.mpi.pixEndpoint</property>
+		<defaultValue>http://localhost:5001/pix</defaultValue>
+		<description>
+			Specifies the endpoint for the Hl7 PIX Message
+		</description>
+	</globalProperty>
+
+	<globalProperty>
+		<property>${project.parent.artifactId}.mpi.pixPort</property>
+		<defaultValue>3700</defaultValue>
+		<description>
+			Specifies port for Hl7 PIX Message
+		</description>
+	</globalProperty>
+
+	<globalProperty>
+		<property>${project.parent.artifactId}.mpi.Hl7implementation</property>
+		<defaultValue>registrationcore.mpiHl7v2HttpSender</defaultValue>
+		<description>
+			Specifies which bean should be used to send the Hl7 Message. Current options are:
+			"registrationcore.mpiHl7v2HttpSender", "registrationcore.mpiHl7v2MllpSender".
+		</description>
+	</globalProperty>
+
+	<globalProperty>
+		<property>${project.parent.artifactId}.mpi.sendingApplication</property>
+		<defaultValue>OpenMRS</defaultValue>
+		<description>
+			Specifies the sending application in the MSH header section of the HL7 message
+		</description>
+	</globalProperty>
+	
+	<globalProperty>
+		<property>${project.parent.artifactId}.birthdateEstimationStartMonth</property>
+		<defaultValue>0</defaultValue>
+		<description>
+			The month from which the birth date is estimated, with January having a value of 0
+		</description>
+	</globalProperty>
+
+	<globalProperty>
+		<property>${project.parent.artifactId}.mpi.sendingFacility</property>
+		<defaultValue>Demo</defaultValue>
+		<description>
+			Specifies sending Facility in the MSH header section of the HL7 message
+		</description>
+	</globalProperty>
+
+	<globalProperty>
+		<property>${project.parent.artifactId}.mpi.receivingApplication</property>
+		<defaultValue>Demo MPI</defaultValue>
+		<description>
+			Specifies the receiving application in the MSH header section of the HL7 message
+		</description>
+	</globalProperty>
+
+
+	<globalProperty>
+		<property>${project.parent.artifactId}.mpi.receivingFacility</property>
+		<defaultValue>OpenEMPI Demo</defaultValue>
+		<description>
+			Specifies sending Facility in the MSH header section of the HL7 message
+		</description>
+	</globalProperty>
+
+	<globalProperty>
+		<property>${project.parent.artifactId}.mpi.pixIdentifierTypeUuidList</property>
+		<defaultValue>05a29f94-c0ed-11e2-94be-8c13b969e334,a5d38e09-efcb-4d91-a526-50ce1ba5011a</defaultValue>
```

Comment:
Are these standard uuids for the identifier type that ship with OpenMRS?
Personally I don't think you need to pick the identifiers to submit based on id type, you should submit all active identifies of the patient

Reply:
Did you address this?

Reply:
yes they are standard uuids for the identifier type that ship with OpenMRS, if we want to do that refactor could we do it as a separate ticket after the merge?